Change JavaScript from Selenium to Poltergeist

bobbywilson0 opened the pull request:
I saw that you wanted to change Selenium to Capybara Webkit. Poltergeist is also webkit driver but runs truly headless instead of a having a Qt dependency. #5

bobbywilson0 committed:
Changed JavaScript driver from Selenium to Poltergeist.

bobbywilson0 committed:
remove unused launchy gem

sbleon commented:
Thanks, @bobbyw! This looks pretty good. There are two things to address before I can merge this:
1. The README should probably say something about the phantomjs dependency (`brew install phantomjs` on my Mac).
2. I'm getting an intermittent test failure with Poltergeist that looks like https://gist.github.com/sbleon/5006206 . This doesn't happen with Selenium.

bobbywilson0 committed:
poltergeist is passing consistently, I added a development section mentioning installing PhantomJS, removed webrat dependency

bobbywilson0 commented:
@sbleon I addressed both of the concerns you mentioned. The issue on the intermittent failures turned out to be webrat matchers, switching this to the capybara has_css matchers fixed the issue. I added a Development section to the README mentioning installing PhantomJS.

bobbywilson0 committed:
remove blank line

bobbywilson0 committed:
remove unnecessary window scope

sbleon commented:
Thanks, @bobbyw!
